### DISCUSSION TITLE
Single component to get the typical stuff, links, emails, phones

### DISCUSSION THREAD

**Discussion by Noitidart** [anantoghosh/react-linkify-it#20] [Ideas]
Thanks very much for this. It was badly needed. Everyone is using the unmaintained react-linkify and it has bugs.

I do miss the ease of react-linkify, where it got all the things you expected it to (links, phone, email - not jira, twitter, mentions etc), and could easily add properties like "target" blank, which you have, but just as example.

Is there a chance for a simpler linke-what-you-expect component?

**Comment by anantoghosh** [comment]
This project provides various components for n number of cases this package can be used for. You can always create your own HOC which has links, phone, email combined into one component. 
The goal of this project is to be highly customization so it's useful in many cases. In my experience, linking requirements have been very specific. For completely general cases other packages already exist.

**Reply by Noitidart** [reply]
Ah I understand. Could you please share some packages you would recommend? I can't find any that are maintained.

**Reply by anantoghosh** [reply]
I couldn't recommend other packages, but there are
- https://www.npmjs.com/package/@mdfe/react-linkify
- https://www.npmjs.com/package/linkify-react
- https://www.npmjs.com/package/react-easy-linkify

**Reply by Noitidart** [reply]
Oh thank you! I had missed linkify-react. I was avoiding all the tasti/react-linkify forks, I used the main one for some time. react-easy-linkify was daunting, the pakcage size was over 1mb. Thanks again brother!
